openedx/features/custom_fields/multiselect_with_other/db/fields.py

```diff
@@ -93,8 +93,11 @@ def to_python(self, value):
             if isinstance(value, list):
                 return value
             elif isinstance(value, string_type):
-                choices_str = value.replace(self.other_delimiter, '')
-                selected_choices = [choice for choice in choices_str.split(',') if choice.strip()]
+                choices_before_and_after_delimiter = value.split(self.other_delimiter)
+                selected_choices = [choice for choice in choices_before_and_after_delimiter[0].split(',')
+                                    if choice.strip()]
+                if len(choices_before_and_after_delimiter) > 1:
+                    selected_choices.append(choices_before_and_after_delimiter[1])
```

Comment:
Still use of indexes isn't readable. It would be great if you please re-consider 👇  suggestion.  Thanks
https://github.com/philanthropy-u/edx-platform/pull/1671#discussion_r455767838

Reply:
I'm not getting how it's violating any standards, we are not concerned with the contents of that indexes whatever will be on those indexes will be added in our resultant list. moreover they are defined in its own scope. So i'm not going to cater this suggestion. If you think this should be added as standard of our code review document please note down this point and we will discuss that while finalizing those.

Reply:
My dear brother It was violating the **READABILITY** standard.